### PR TITLE
Add sub page

### PR DESCRIPTION
**Pull request by MaciejSurowiec**
just to test how it looks

### CODE REVIEW

File: BeerDiplomaWebpage/src/app/auth/auth.component.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+@Component({
+  selector: 'app-auth',
+  templateUrl: './auth.component.html',
+  styleUrls: ['./auth.component.css']
+})
+export class AuthComponent implements OnInit {
+  authType: String = '';
+  title: String = '';
+  jebanyan: Boolean = true;
```
Comment (szwedosz) [inline BeerDiplomaWebpage/src/app/auth/auth.component.ts]: what is this? xD

File: BeerDiplomaWebpage/src/app/home/home.component.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<h1 class="center">Strona Glowna</h1>
```
Comment (szwedosz) [inline BeerDiplomaWebpage/src/app/home/home.component.html:1]: główna

File: BeerDiplomaWebpage/src/styles.css [hunk 3]
```diff
@@ -1 +1,61 @@
-/* You can add global styles to this file, and also import other style files */
+.form-group{
+border-style: none;
+}
+
+.form{
```
Comment (szwedosz) [inline BeerDiplomaWebpage/src/styles.css]: style - you are missing " " before { everywhere

File: BeerDiplomaWebpage/src/styles.css [hunk 4]
```diff
@@ -1 +1,61 @@
-/* You can add global styles to this file, and also import other style files */
+.form-group{
+border-style: none;
```
Comment (szwedosz) [inline BeerDiplomaWebpage/src/styles.css]: incorret indent

File: BeerDiplomaWebpage/src/app/auth/auth.component.ts (hunk 1 above)
Comment (MaciejSurowiec) [reply BeerDiplomaWebpage/src/app/auth/auth.component.ts]: It’s just testing value, and it supposed to be named „jebanyangular” xd

**Pull request merged**